### DISCUSSION TITLE
Discussion on impacts of Open Data Legislation

### DISCUSSION THREAD

**Discussion by SnaggyD** [iodepo/OceanDecade-dsig#26] [General]
This is discussion about the impact of the Open Data Directive (2019) from the EU. It was adopted by all member states in 2019 and directs that all publicly funded data (particularly environment observations) are published as Open Data .

**Comment by pbuttigieg** [comment]
One complexity on this for scientific data is the wrangling around embargos. (xref https://github.com/iodepo/OceanDecade-dsig/issues/3)

I would say that most scientists do not want (or rather, cannot have if they wish to be competitive in the existing crediting system) their data out in the open before they've secured credit for an academic publication. Most academic institutes (all that I've encountered) will largely ignore high-quality data/software etc contributions to science, if there's no high-impact ([whatever that means](https://www.science.org/content/article/hate-journal-impact-factors-new-study-gives-you-one-more-reason)) paper accompanying it and it seems none have the courage to change.

Some funders/institutes thus allow an embargo period, however this is preventing vital data and information from being available to all to face urgent challenges. This is an ethical question, tied to a crediting system incompatible with modern data needs by science and society.
